.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [ 1.15.x, 1.16.x]
+        go-version: [ 1.15.x, 1.16.x, 1.17.x ]
```

Comment:
How do we feel about dropping 1.15? Feel like we should try to align to the upstream policy when possible: 

https://golang.org/doc/devel/release#policy

Double checked and the Snap is already on 1.16:

https://github.com/digitalocean/doctl/blob/283ec460997e86b99a09a2bd4ba223636eece782/dockerfiles/Dockerfile.snap#L49

Reply:
On the other hand, it doesn't hurt to know if it passes or not... Maybe just mark as not required?